### PR TITLE
Fixed naming and catch all cert fetching

### PR DESCRIPTION
**Pull request by zdmytriv**
https://app.clubhouse.io/vgs/story/57920/fix-naming-in-nginx-ssl-router

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -14,7 +14,6 @@ cert_mgr_url: http://certificates.verygoodsecurity.io
 ssl_router_tld: verygoodproxy.io
 ssl_router_upstream_overrides_tnt: {}
 ssl_router_upstream_overrides_chn: {}
-ssl_router_data_env: "sandbox"
```
Comment (zdmytriv) [inline defaults/main.yml]: no defaults intentionally

File: tasks/main.yml [hunk 2]
```diff
@@ -89,7 +89,7 @@
   template: src=templates/verygoodproxy.conf.j2 dest={{ ssl_router_nginx_config_dir }}verygoodproxy.conf mode=0400
 
 - name: lookup catch all certificate
-  shell: "aws s3 ls s3://{{ ssl_router_s3_bucket }}/ssl/ | grep '{{ ssl_router_data_env }}.{{ ssl_router_tld }}' | awk '{print $4}'"
+  shell: "aws s3 ls s3://{{ ssl_router_s3_bucket }}/ssl/ | grep '{{ ssl_router_host_prefix }}.{{ ssl_router_tld }}' | awk '{ if($3 >0) print $4 }'"
```
Comment (zdmytriv) [inline tasks/main.yml]: fixed `awk`. It will allow to avoid issues with dirs that matches pattern.

File: defaults/main.yml (hunk 1 above)
Comment (max-lobur) [reply defaults/main.yml]: leave it "", and assert non ""

File: tasks/main.yml [hunk 3]
```diff
@@ -88,8 +88,11 @@
 - name: add catch all
   template: src=templates/verygoodproxy.conf.j2 dest={{ ssl_router_nginx_config_dir }}verygoodproxy.conf mode=0400
 
+- name: check that "ssl_router_host_prefix" is set
+  assert: { that: "{{ ssl_router_host_prefix|length > 0}}" }
```
Comment (zdmytriv) [inline tasks/main.yml]: added assert

Comment (max-lobur) [reply tasks/main.yml]: ++

**Pull request merged**